apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/SettingsActivity.java

```diff
@@ -82,6 +88,14 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
                 .commit();
     }
 
+    public static String defaultLogUrlFunction(String startType){
+        if(StartType.valueOf(startType) == StartType.TARGET){
+            return oneCollectorLogUrl;
+        }else{
+            return otherLogUrl;
```

Comment:
You can inlined the if-else using ternary operation `? :` to make it more readable.